index.html

```diff
@@ -323,24 +334,77 @@ <h3 class="custom-tags-font default-custom-font about-tag-color">Selenium</h3>
+    <section id="contact" class="section-contact-container flex-container-column">
+      <h2 class="legend-font-desktop primary-text-custom-font">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xrgvannb" method="POST" class="form-container flex-container-column">
+        <h2 class="legend-font primary-text-custom-font">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <label for="user_name"></label>
+        <input class="form-name default-input-style default-custom-font" type="text" id="user_name" name="user_name"
+          placeholder="Enter your name" maxlength="30" required />
+        <label for="user_email"></label>
+        <input class="form-email default-input-style default-custom-font" type="email" id="user_email" name="user_email"
+          placeholder="Enter your e-mail" required />
+        <label for="user_message"></label>
+        <div id="email_message" class="email-message default-custom-font disabled">E-mail need to be passed in lowercase
+          letters only </div>
+        <textarea class="default-textarea-style default-custom-font" id="user_message" name="user_message"
+          placeholder="Enter text here" maxlength="500" required></textarea>
+        <button class="custom-button" type="submit">Get in Touch</button>
+      </form>
+    </section>
+
+    <!--Added by Sandeep Ghosh, code Partner mailsg0912@gmail.com-->
+
+    <!-- Footer section-->
+    <footer class="desktop-footer flex-container-row ">
+      <ul class="desktop-footer-list headline social-media-icons align-center form-links-icon">
+        <li class="headline-item">
+          <a class="twitter" href="https://twitter.com/MandeAlemayehu" title="Contact-me: Twitter link"><img
+              src="./css/img/headline/icons/twitter.png" alt="My twitter"></a>
+       </li>
+        <li class="headline-item">
+          <a class="linkedin" href="https://www.linkedin.com/in/habtamu-alemayehu-b90367101/" alt="My linkedin"><img
+              src="./css/img/headline/icons/linkedin.png" /></a>
+        </li>
+        <li class="headline-item">
+          <a class="medium" href="https://medium.com/@habtamualemayehu16" title="Contact-me: Medium link"><img
+              src="./css/img/headline/icons/medium.png" alt="My medium"></a>
+        </li>
+        <li class="headline-item">
+          <a class="github" href="https://github.com/Benawi" title="Contact-me: Github link"><img
+              src="./css/img/headline/icons/github.png" alt="My github"></a>
+        </li>
+        <li class="headline-item">
+          <a class="angellist" href="https://venture.angellist.com/v/login" title="Contact-me: AngelList link"><img
+              src="./css/img/headline/icons/angellist.png" alt="My angellist"></a>
+        </li>
+      </ul>
+    </footer>
```

Comment:
- To match the Figma design:
   - [ ] Kindly add the small background on the right. 
   - [ ] Kindly add your email between the `Get in touch` button and social media icons.
   - [ ] Please center the social media icons on the middle.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/228567087-b185e04d-1037-44b9-b765-6bceb0c442f7.png) | ![image](https://user-images.githubusercontent.com/24703171/228566636-56fe4624-981a-42c9-8a14-5c895568ae08.png) |